src/SwiftRuntimeLibrary/src/TypeDatabase.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Xml;
+
+namespace SwiftRuntimeLibrary
```

Comment:
```suggestion
namespace Swift.Runtime
```

Reply:
Do you think we should change the .csproj name to `SwiftRuntime.csproj`/`SwiftRuntime.Tests.csproj`?

Reply:
Maybe Swift.Runtime[.Test].csproj?

Reply:
Changed to Swift.Bindings and Swift.Runtime for clarity.